README.md

```diff
@@ -611,6 +610,8 @@ FLAGS
   --noEmoji               When set to true, disables displaying emoji in output.
   --profileId=<value>     (required) Profile Id in format [scope/](optional)[name]
   --providerName=<value>  (required) Name of the provider. This argument is used to publish a map or a provider.
+  --switch                After publish user will be asked if he wants to switch to remote version of
```

Comment:
```suggestion
  --switch                After publish user will be asked if they want to switch to remote version of
```

Reply:
Alternative: After publish ask whether to switch to the remote version of profile/map/provider